aasemble/django/apps/api/v1/serializers.py

```diff
@@ -53,7 +53,7 @@ class SnapshotSerializer(serializers.HyperlinkedModelSerializer):
 
     class Meta:
         model = mirrorsvc_models.Snapshot
-        fields = ('self', 'timestamp', 'mirrorset')
+        fields = ('self', 'timestamp', 'mirrorset', 'tag')
```

Comment:
Oh, and I don't think we should add more features to the v1 API. Please only add this to the v2 API.